src/data/nat/modeq.lean

```diff
@@ -2,13 +2,25 @@
 Copyright (c) 2017 Mario Carneiro. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mario Carneiro
-
-Modular equality relation.
 -/
 import data.int.gcd
 import tactic.abel
 import data.list.rotate
+/-
+# nat.modeq -- congruences modulo a natural on ℕ.
```

Comment:
```suggestion
# Congruences modulo a natural number
```

Reply:
I wanted to make it clear that a and b were naturals as well as n. I'm happy to change.

Reply:
I've just looked at data.int.modeq and I now see that there all three of a,b,n are integers. For some reason I was expecting n to be a nat. I'll go with your suggestion.